content/02.main-text.md

```diff
@@ -204,13 +230,32 @@ OpenTimestamps minimizes fees by encoding many timestamps into a single Bitcoin
 We find that this asynchronous design with timestamps precise to the day is suitable for the purposes of scientific writing.
 
+### Reproducible manuscripts
+
 Manubot and its dependencies are free of charge and largely open source.
 It does rely on gratis services from two proprietary platforms: GitHub and Travis CI.
 Fortunately, lock-in to these services is minimal, and several substitutes already exist.
 Manubot provides a substantial step towards end-to-end document reproducibility, where every figure or piece of data in a manuscript can be traced back to its origin [@url:https://elifesciences.org/for-the-press/e6038800/elife-supports-development-of-open-technology-stack-for-publishing-reproducible-manuscripts-online] and is well suited for preserving provenance.
 For example, figures can be specified using versioned URLs that refer to the code that created them.
 In addition, manuscripts can be templated, so that numerical values or tables are inserted directly from the repository that created them.
-An [example repository](https://github.com/manubot/rootstock) demonstrates Manubot's features and serves as a template for users to write their own manuscript with Manubot.
+
+### Getting started
+
+An example repository at <https://github.com/manubot/rootstock> demonstrates Manubot's features and serves as a template for users to write their own manuscripts with Manubot.
+The [current setup process](https://github.com/manubot/rootstock/blob/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/SETUP.md) includes cloning the Rootstock repository, rebranding it to the user's manuscript, and configuring continuous integration.
+The setup process is complex, but most only be performed once per manuscript.
+Upgrading manuscripts is also possible by pulling the latest commits from Rootstock, but sometimes involves resolving Git conflicts.
+
+Contributing to an existing manuscript is less technical and can be performed entirely through GitHub's web interface, as discussed in the [Contribution Workflow](#contribution-workflow) section and demonstrated in Video S1.
+Interested readers can practice editing an existing demo manuscript at <https://github.com/manubot/try-manubot>.
+
+At the 2019 _Pacific Symposium on Biocomputing_, we led a working group where 17 conference participants contributed to a [demo manuscript](https://git.dhimmel.com/psb-manuscript/).
+Based on this experience, we believe most computational scholars have the expertise to contribute to a Manubot manuscript.
+Proficiency with Manubot requires familiarity with Markdown, Git, GitHub, and continuous integration.
+While these tools do present a barrier to entry, they are also highly applicable outside of Manubot and increasingly part of the standard curriculum for computational scholars.
+For example, Markdown is used for documenting Jupyter and Rmarkdown notebooks.
```

Comment:
I was wrong before, it should be `R Markdown` notebooks per https://rmarkdown.rstudio.com/